book/book.toml

```diff
@@ -0,0 +1,6 @@
+[book]
+authors = ["Bilal Elmoussaoui"]
```

Comment:
Please use the same that we use everywhere ("gtk-rs developers").

Reply:
Ah sorry, completely missed this one